src/elements/selection-expansion-panel/selection-expansion-panel.ts

```diff
@@ -169,9 +169,19 @@ export class SbbSelectionExpansionPanelElement extends SbbHydrationMixin(LitElem
 
     this._checked = input.checked;
     this._disabled = input.disabled;
+    this._setDataSize(input);
```

Comment:
Proposal: 
would an AttributeObserver on the inner `checkbox/radio-button panel` be enough to cover every case?
With that, we would have a single point where the `data-size` is set and we could remove the dependency on the `selection-expansion-panel` from the `checkbox/radio-button group` and the `checkbox/radio-button panel`

Reply:
It would be a bottom-up approach (the observer should be placed in the inner panel); i'll create a PoC from this branch

Reply:
@TomMenga 
https://github.com/sbb-design-systems/lyne-components/compare/feat/selection-panel-size-s-bottomup?expand=1

Reply:
mmm, that's not what I meant :/ I may have explained it poorly. Let's have a call after the daily

Reply:
Example updated, same link